lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -305,16 +340,35 @@ class CategoryRenderer {
 
     const metricAudits = category.audits.filter(audit => audit.group === 'perf-metric');
     const metricAuditsEl = this._renderAuditGroup(groups['perf-metric']);
+    const timelineContainerEl = this._dom.createChildOf(metricAuditsEl, 'div',
+        'lh-timeline-container');
+    const timelineEl = this._dom.createChildOf(timelineContainerEl, 'div', 'lh-timeline');
+
+    let perfTimelineScale = 0;
+    metricAudits.forEach(audit => {
+      if (Number.isFinite(/** @type {number} */ (audit.result.rawValue))) {
```

Comment:
are you worried about  `NaN`s or `Infinity`s? Checking for `undefined` would be a lot clearer here if not